go/service/user.go

```diff
@@ -475,6 +475,8 @@ func (h *UserHandler) proofSuggestionsHelper(mctx libkb.MetaContext) (ret []Proo
 		switch displayConfig.Key {
 		case "zcash.t", "zcash.z", "zcash.s":
 			altKey = "zcash"
+		case "bitcoin":
+			altKey = "btc"
```

Comment:
this this affect ident2?

Reply:
Nope this only affects the ProofSuggestions RPC which was not used prior to id3